catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/ImageAttribute.java

```diff
@@ -211,92 +211,92 @@
             "ISCRSN",
             segment -> segment.getSecurityMetadata()
                     .getClassificationReason(),
             BasicTypes.STRING_TYPE,
             ATTRIBUTE_NAME_PREFIX);
 
     public static final ImageAttribute IMAGE_SECURITY_SOURCE_DATE = new ImageAttribute(
-            "imageSecuritySourceDate",
+            "image-security-source-date",
             "ISSRDT",
             segment -> segment.getSecurityMetadata()
                     .getSecuritySourceDate(),
             BasicTypes.STRING_TYPE,
             ATTRIBUTE_NAME_PREFIX);
 
     public static final ImageAttribute IMAGE_SECURITY_CONTROL_NUMBER = new ImageAttribute(
-            "imageSecurityControlNumber",
+            "image-security-control-number",
             "ISCTLN",
             segment -> segment.getSecurityMetadata()
                     .getSecurityControlNumber(),
             BasicTypes.STRING_TYPE,
             ATTRIBUTE_NAME_PREFIX);
 
-    public static final ImageAttribute PIXEL_VALUE_TYPE = new ImageAttribute("pixelValueType",
+    public static final ImageAttribute PIXEL_VALUE_TYPE = new ImageAttribute("pixel-value-type",
             "PVTYPE",
             segment -> segment.getPixelValueType()
                     .name(),
             BasicTypes.STRING_TYPE,
             ATTRIBUTE_NAME_PREFIX);
 
     public static final ImageAttribute ACTUAL_BITS_PER_PIXEL_PER_BAND = new ImageAttribute(
-            "actualBitsPerPixelPerBand",
+            "actual-bits-per-pixel-per-band",
             "ABPP",
             ImageSegment::getActualBitsPerPixelPerBand,
             BasicTypes.INTEGER_TYPE,
             ATTRIBUTE_NAME_PREFIX);
 
     public static final ImageAttribute PIXEL_JUSTIFICATION =
-            new ImageAttribute("pixelJustification",
+            new ImageAttribute("pixel-justification",
                     "PJUST",
                     segment -> segment.getPixelJustification()
                             .name(),
                     BasicTypes.STRING_TYPE,
                     ATTRIBUTE_NAME_PREFIX);
 
     public static final ImageAttribute IMAGE_COORDINATE_REPRESENTATION = new ImageAttribute(
-            "imageCoordinateRepresentation",
+            "image-coordinate-representation",
             "ICORDS",
             segment -> segment.getImageCoordinatesRepresentation()
                     .name(),
             BasicTypes.STRING_TYPE,
             ATTRIBUTE_NAME_PREFIX);
 
     public static final ImageAttribute NUMBER_OF_IMAGE_COMMENTS = new ImageAttribute(
-            "numberOfImageComments",
+            "number-of-image-comments",
             "NICOM",
             segment -> segment.getImageComments()
                     .size(),
             BasicTypes.INTEGER_TYPE,
             ATTRIBUTE_NAME_PREFIX);
 
-    public static final ImageAttribute IMAGE_COMMENT_1 = new ImageAttribute("imageComment1",
+    public static final ImageAttribute IMAGE_COMMENT_1 = new ImageAttribute("image-comment1",
```

Comment:
we did `image-identifier-2` and `image-identifier-1`, do we want to make these `image-comment-1`?